usr/sbin/p11sak/p11sak.c

```diff
@@ -2781,7 +2781,8 @@ static CK_RV parse_file() {
     // open and parse file
     if (file_loc != NULL) {
         if ((fp = fopen(file_loc, "r")) == NULL) {
-            fprintf(stderr, "Failed to load %s: %s\n", file_loc, strerror(errno));
+            fprintf(stderr, "Cannot load config file from env variable %s: %s\n", file_loc, strerror(errno));
+            fprintf(stderr, "Printing of vendor attributes not available.\n");
```

Comment:
I would not call this 'vendor' attributes, but maybe 'custom' attributes? Attributes like CKA_MODULUS_BITS and CKA_MODULUS are in fact no vendor attributes, just attributes that are normally not printed....

Reply:
OK, done.